pages/trees/[id].tsx

```diff
@@ -97,7 +97,7 @@ const InfoList: FC<{
       <DataListItem
         title={t(`treeView.infoList.waterSupply.label`)}
         subtitle={t(`treeView.infoList.waterSupply.hint`)}
-        value={
+        valueLabel={
```

Comment:
Based on the changes this component should receive `iconType` and `iconValue` for each data item. At the moment the sidebar looks broken, now that we are only displaying the small text value without the visualizations.

Reply:
@dnsos I updated the PR based on your comments. Cannot implement into `page/trees/[id].tsx` though, because I cannot get trees on the map to be rendered on localhost. Map is shown without the dots, due to cross-origin-error.
![image](https://user-images.githubusercontent.com/292137/217192222-6177f42d-7092-47de-b084-78ac949c7a0f.png)

Cannot check on [production](https://baumblick-frontend-aoia0oeai-technologiestiftung1.vercel.app/trees) either, because there's an AJAXError:
![image](https://user-images.githubusercontent.com/292137/217192724-bf9f11f4-2c85-42ce-98f0-cb8e8385dd40.png)

Reply:
I have added the visualizations to the sidebar now, so resolving this.